eos_data_distribution/producers/usb.py

```diff
@@ -35,36 +37,84 @@
 logging.basicConfig(level=logging.INFO)
 logger = logging.getLogger(__name__)
 
+
+# Timeout after which this daemon will exit if no interesting mounts exist.
+IDLE_TIMEOUT = 30  # seconds
```

Comment:
Only if it’s going to be used by more than one module, which at the moment is not the case. We can always move it across later, but for the moment I think it makes sense for it to stay in the file where it’s used.